frontend/src/StoreState.ts

```diff
@@ -0,0 +1,13 @@
+import Route from './structures/route';
+import Vehicle from './structures/vehicle';
+import Stop from '@/structures/stop';
```

Comment:
So I like the `@` vs. relative paths... all of the imports should probably do the same thing